apps/desktop/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"extends": "./.svelte-kit/tsconfig.json",
 	"compilerOptions": {
-		"target": "es6",
+		"target": "ES2022",
```

Comment:
When targeting es6, the `$state()` and `$derived()` statements weren't allowed in classes due to how ts transformed them into declarations inside the constructor.